torch/ao/quantization/fx/_lower_to_native_backend.py

```diff
@@ -566,8 +563,11 @@ def _lower_dynamic_weighted_ref_module(model: QuantizedGraphModule):
         # remove q - dq node
         dq_node.replace_all_uses_with(input_dynamic_q_node)
         model.graph.erase_node(dq_node)
-        input_dynamic_q_node.replace_all_uses_with(input_dynamic_q_node.args[0])
-        model.graph.erase_node(input_dynamic_q_node)
+        dynamic_remove_set.add(input_dynamic_q_node)
+
+    for node in dynamic_remove_set:
+        node.replace_all_uses_with(node.args[0])
+        model.graph.erase_node(node)
```

Comment:
we can rely on dead code elimination to remove this I think: https://github.com/pytorch/pytorch/blob/master/torch/ao/quantization/fx/_lower_to_native_backend.py#L966